### PR TITLE
Remaining jet pT eta phi N variables implementation

### PR DESCRIPTION
**Pull request by lucastorterotot**
In this PR, the computation of the "remaining jets" summed momentum and the number of jets in this variable is done.
Feel free to propose ways to improve this code, especially to avoid some duplication of lines (see changes).

### CODE REVIEW

File: KappaAnalysis/interface/Producers/ValidJetsProducer.h [hunk 1]
```diff
@@ -187,6 +187,66 @@ class ValidJetsProducerBase: public KappaProducerBase, public ValidPhysicsObject
...
+                            remaining_jets_p4 += (*jet)->p4 ;
+                        };
+                    };
+                };
+                return remaining_jets_p4.Eta() ;
+            });
+
+        LambdaNtupleConsumer<KappaTypes>::AddFloatQuantity("remainingJetPhi", [settings](KappaEvent const& event, KappaProduct const& product) {
+                ROOT::Math::LorentzVector<ROOT::Math::PxPyPzE4D<double> > remaining_jets_p4 (0, 0, 0, 0);
+                if (product.m_validJets.size() >= 3) {
```
Comment (harrypuuter) [inline KappaAnalysis/interface/Producers/ValidJetsProducer.h:226]: maybe you should set a better default value than 0, usually we use -10 here

Comment (harrypuuter) [reply KappaAnalysis/interface/Producers/ValidJetsProducer.h:226]: for all the variables so pt,eta,phi

**Review by harrypuuter — CHANGES_REQUESTED** [review]
rest looks fine

File: KappaAnalysis/interface/Producers/ValidJetsProducer.h (hunk 1 above)
Comment (lucastorterotot) [reply KappaAnalysis/interface/Producers/ValidJetsProducer.h:226]: Does [this commit](https://github.com/KIT-CMS/Artus/pull/25/commits/934be376eef5144d932d961ae59700b8a1af68ea) makes it correct for you?

Comment (harrypuuter) [reply KappaAnalysis/interface/Producers/ValidJetsProducer.h:226]: I just remembered, for consistency you can use the defaults defined here: https://github.com/KIT-CMS/Artus/blob/reduced_trigger_objects/Utility/src/DefaultValues.cc

Comment (lucastorterotot) [reply KappaAnalysis/interface/Producers/ValidJetsProducer.h:226]: Ok I updated the code accordingly and tested it.
Note that, if there are more than 2 jets but the remaining ones have pT lower than the `GetJetOfflineLowerPtCut` threshold then you can end up with `Njets_r == 0` and `jpt_r == 0` as well with the current code, as can be seen from [this condition](https://github.com/lucastorterotot/Artus/blob/23dbdec2ac79987c44cf5cd3ad605a983d034a76/KappaAnalysis/interface/Producers/ValidJetsProducer.h#L200-L202).
Would it be better to check this pT threshold only after the sum to only have `jpt_r == -10` for `Njet_r == 0`?

**Review by harrypuuter — APPROVED** [review]

**Pull request merged**